tests/test_main_startup_blocking.py

```
@@ -0,0 +1,230 @@
+    return main, client_mod
+
+
+async def _teardown_main(main, client_mod):
+    for name in ("_startup_connect_task", "_cache_warm_task", "_runtime_bootstrap_task"):
+        task = getattr(main, name, None)
+        if task is not None and not task.done():
+            task.cancel()
+            try:
+                await task
+            except (asyncio.CancelledError, Exception):
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)